tests/cms/views/regions/delete_region.py

```diff
@@ -0,0 +1,122 @@
+    assert (
+        get_user_model().objects.filter(is_superuser=False, is_staff=False).count() > 0
+    )
+
+    # Deletion of Augsburg will not be possible due to a cloned page in Nurnberg.
+    # We want to test this is in the test below. In this test we want to make sure this is the only reason why deleting is not possible.
+    cloned_page = Page.objects.filter(id=CLONED_PAGE)
+    cloned_page.delete()
+
+    # We want to test with existing data, as we want to make sure as many dependencies as possible are covered
+    for region in Region.objects.all():
+        delete_region = reverse("delete_region", kwargs={"slug": region.slug})
+        response = client.post(delete_region, data={})
+
+        if role == ANONYMOUS:
+            assert response.status_code == 302
+            assert (
+                response.headers.get("location")
+                == f"{settings.LOGIN_URL}?next={delete_region}"
+            )
+            return
+
+        if role not in [CMS_TEAM, SERVICE_TEAM, ROOT] and not ANONYMOUS:
+            assert response.status_code == 403
+            continue
+
+        if role in [CMS_TEAM, SERVICE_TEAM, ROOT]:
+            assert response.status_code == 302
+            redirect = response.headers.get("location")
+            response = client.get(redirect)
+            assert "Region wurde erfolgreich gelöscht" in response.content.decode(
+                "utf-8"
+            )
+
+    if role in [CMS_TEAM, SERVICE_TEAM, ROOT]:
+        assert Region.objects.count() == 0
+        # No users without region
+        assert (
+            get_user_model()
+            .objects.filter(is_superuser=False, is_staff=False, regions=None)
+            .count()
+            == 0
+        )
+        # No push notification without region
+        assert PushNotification.objects.filter(regions=None).count() == 0
+
+
+@pytest.mark.django_db
+def test_deleting_mirrored_region_is_unsucessful(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+) -> None:
+    client, role = login_role_user
+
+    augsburg = Region.objects.get(slug="augsburg")
+    cloned_page = Page.objects.filter(id=CLONED_PAGE)
+    assert cloned_page.exists()
+
+    delete_region = reverse("delete_region", kwargs={"slug": augsburg.slug})
+    response = client.post(delete_region, data={})
+
+    if role == ANONYMOUS:
+        assert response.status_code == 302
+        assert (
+            response.headers.get("location")
+            == f"{settings.LOGIN_URL}?next={delete_region}"
+        )
+
+    if role not in [CMS_TEAM, SERVICE_TEAM, ROOT] and not ANONYMOUS:
+        assert response.status_code == 403
+
+    if role in [CMS_TEAM, SERVICE_TEAM, ROOT]:
+        assert response.status_code == 302
+        redirect = response.headers.get("location")
+        response = client.get(redirect)
+        assert (
+            "Die Region konnte nicht gelöscht werden, weil die folgenden Seiten in anderen Region gespiegelt werden:"
```

Comment:
There is a typo here with "Region" instead "Regionen", but it's also in the translation file so I think it's probably just copied from there?

Reply:
Hmm, I think singular is correct here, as `delete_region` delets actually one single region 👀

Reply:
```suggestion
            "Die Region konnte nicht gelöscht werden, weil die folgenden Seiten in anderen Regionen gespiegelt werden:"
```
I think @jarlhengstmengel was referring to the second occurrence, which should actually be plural.

Reply:
@jarlhengstmengel @david-venhoff 
Ah 🙈 Yes, it must be "Regionen" there. #3302